src/main/java/com/socialbox/controllers/UserController.java

```diff
@@ -2,10 +2,14 @@
 
 import com.socialbox.model.User;
 import com.socialbox.service.UserService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
```

Comment:
Imports ending with `.*` means that the whole while is imported. Try to import only the things required.